lib/mongo/operation/write/create_index.rb

```diff
@@ -42,37 +42,14 @@ module Write
       #
       # @since 2.0.0
       class CreateIndex
-        include Executable
+        include GLEable
+        include WriteCommandEnablable
```

Comment:
Maybe for grammatical purposes `WriteCommandEnabled` might be more sane here. :)